include/flatbuffers/flatbuffers.h

```diff
@@ -559,9 +559,11 @@ static inline bool StringLessThan(const char *a_data, uoffset_t a_size,
 
 struct String : public Vector<char> {
   const char *c_str() const { return reinterpret_cast<const char *>(Data()); }
+#ifndef FLATBUFFERS_NO_STL
```

Comment:
Please indent any preprocessor directives that are inside an indented (class or function) scope, and add `// clang-format off` directives as needed (see rest of this header).

Reply:
Done. Tried to keep the clang-format off in as minimal a scope as possible, but let me know if there's improvements that can be made.